autoload/reply/repl/stack_ghci.vim

```diff
@@ -0,0 +1,34 @@
+let s:repl = reply#repl#base('stack ghci')
+
+function! s:repl.executable() abort
+  return "stack"
+endfunction
+
+function! s:repl.get_command() abort
+    return [self.executable(), 'ghci']
+           \+ self.default_cmdopts()
+endfunction
+
+" TODO: It's dirty hack but i didn't find other solution
+"  I tryed to play with `+ self.get_var('command_options', ['--no-load'])`
+"  in the get_command() function but it didn't work because
+"  'command_options' always empty!
```

Comment:
What does 'empty' mean? `self.get_var('command_options', ['--no-load'])` should return `['--no-load']` when you don't set neither `g:reply_repl_stack_ghci_command_options` nor `b:reply_repl_stack_ghci_command_options`.

Reply:
You wrote nice code but I  faced with the problem to pass `--no-load` flag to `stack ghci` **only for the case** when user **doesn't input explicitly command** `Repl stack_ghci some_cmdopts`. Without `--no-load` `stack` will load to `ghci` all project modules. But If user explicitly types the command the flag should not be passed.

If i found right solution you should delete `TODO`.